client/src/components/modals/Modal.tsx

```diff
@@ -295,6 +319,17 @@ export const Modal: React.FC<ModalProps> = ({
         >
           {ctaButtonTitle}
         </Button>
+
+        {transactionAddress?.length ? (
+          <Link
+            href={`https://goerli.etherscan.io/tx/${transactionAddress}`}
```

Comment:
Can you add the block scan link to `SmartContractContext` and pull that in here?